deepmd/pt/model/task/property.py

```
@@ -82,6 +82,7 @@ def __init__(
         bias_atom_p: Optional[torch.Tensor] = None,
         intensive: bool = False,
         bias_method: str = "normal",
+        property_name: Union[str, list] = "property",
```

Ruff (0.8.2)

Undefined name `Union`

(F821)